Library/Formula/sqlitebrowser.rb

```diff
@@ -14,12 +12,18 @@ class Sqlitebrowser < Formula
     sha1 "b633087ab28885f058bc5bcf24ca834828d1c894" => :mountain_lion
   end
 
-  depends_on 'qt'
-  depends_on 'cmake' => :build
-  depends_on 'sqlite' => 'with-functions'
+  depends_on "qt"
+  depends_on "cmake" => :build
+  depends_on "sqlite" => "with-functions"
+  depends_on "sqlcipher" => :optional
 
   def install
     system "cmake", ".", *std_cmake_args
-    system 'make install'
+    system "make", "install"
+  end
+
+  test do
+    shell_output("#{bin}/sqlitebrowser -h")
+    assert_equal 0, $?.exitstatus
```

Comment:
Looks like sqlitebrowse "captures" output error (maybe because it works wit qt?):

(without errors opening fake files)

alex@vostok:/tmp/homebrew/Library/Formula(sqlitebrowser)$ brew test --verbose sqlitebrowser
Testing sqlitebrowser
==> /usr/local/Cellar/sqlitebrowser/3.5.0/bin/sqlitebrowser -s /private/tmp/sqlitebrowser-CGzLxm/fake.sql -q /private/tmp/sqlitebrowser-CGzLxm/fake.db
The file /private/tmp/sqlitebrowser-CGzLxm/fake.sql does not exist
Invalid option/non-existant file: /private/tmp/sqlitebrowser-CGzLxm/fake.db
alex@vostok:/tmp/homebrew/Library/Formula(sqlitebrowser)$